.circleci/config.yml

```diff
@@ -65,16 +65,38 @@ jobs:
           command: |
             # show env variables but not the AWS secrets!
             env | egrep -v AWS_
```

Comment:
I know this wasn't part of this patch, but it would probably be a good idea to remove this line in case a sensitive env var that doesn't start with `AWS_` is added in the future.

Reply:
The line was there for troubleshooting purpose. I'm OK with removing it.